src/WebServer/Rest/RestControllerMethodInfo.cs

```diff
@@ -20,9 +20,9 @@ internal enum TypeWrapper
         private readonly IEnumerable<Type> _validParameterTypes;
 
         private readonly UriParser _uriParser;
-        private readonly ParsedUri _matchUri;
         private readonly IEnumerable<ParameterValueGetter> _parameterGetters;
 
+        internal ParsedUri MatchUri { get; }

```

Comment:
It's used by the UniqueMatchUriAndVerbRestControllerMethodInfoComparer, couldn't figure out a way to do it in any other way...